src/test/resources/features/Scoring/Student_Question/StudentQuestion.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Student Question
+
+  Background:
+    Given user prepare auth request
+    When user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare quiz request with batchCode "futur3"
+    And user hit get all quiz endpoint
+    And user get first quiz id and store id
+    And user get first quiz and store id
+    And user prepare student question request with batchCode "futur3"
```

Comment:
Also, does this background require batch with batchCode `futur3` to exist prior to running this feature's tests?